GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerTestCase/RepairTests.cs

```diff
@@ -13,6 +13,14 @@ namespace GVFS.FunctionalTests.Tests.EnlistmentPerTestCase
     [Category(Categories.MacTODO.M4)]
     public class RepairTests : TestsWithEnlistmentPerTestCase
     {
+        [TestCase]
+        public void NoFixesNeeded()
+        {
+            this.Enlistment.UnmountGVFS();
+
+            this.Enlistment.Repair();

```

Comment:
Do you know if the repair without the `--confirm` would have caught this as well?  I was thinking that for each of our tests in this class would should run the repair without `--confirm` before running the confirm version to catch other possible code paths.

Reply:
@kewillford , yes without --confirm we still get the error.  It's up to us if we want to do another run without the flag.  My only concern would be the length of time to run the tests, but if it's a test worth doing we should do it!

Reply:
I was thinking we could do it in the test cases we already have not adding new test cases.
Run repair without `--confirm` then run with it in the same test case.  Hopefully this wouldn't add to much time.